sceptre/plan/actions.py

```diff
@@ -526,6 +526,17 @@ def execute_change_set(self, change_set_name):
         :rtype: str
         """
         self._protect_execution()
+        change_set = self.describe_change_set(change_set_name)
+        status = change_set.get("Status")
+        reason = change_set.get("StatusReason")
+        if status == "FAILED" and "submitted information didn't contain changes" in reason:
```

Comment:
This looks good to me.   I'm just wondering why you used `StatusReason` instead of the `ExecutionStatus` for this check?  Looks like checking "ExecutionStatus": "UNAVAILABLE" would work as well and it might be more consistent than the message from StatusReason?